project-canvas/src/components/backlog-view/Issue/DetailView.tsx

```diff
@@ -56,180 +75,123 @@ export function DetailView({
-            handleDoubleClick={() => setShowInputEle(true)}
-            value={defaultdescription}
-          />
-
-          <Text color="dimmed" mb="sm">
-            Child Issues
-          </Text>
-          {subtasks.length !== 0 ? (
-            <Paper mb={30} withBorder>
+          <ScrollArea.Autosize
+            maxHeight="70vh"
+            mr="xs"
+            sx={{ minWidth: "260px" }}
+          >
+            <Text color="dimmed" mb="sm">
+              Description
+            </Text>
+            <Description issueKey={issueKey} description={description} />
+            <Text color="dimmed" mb="sm">
+              Child Issues
+            </Text>
+            <Paper mb="lg" mr="sm">
               <Stack spacing="xs">
                 {subtasks.map((subtask) => (
-                  <Paper
-                    withBorder
-                    key={subtask.id}
-                    p={5}
-                    sx={(theme) => ({ display: "flex", gap: theme.spacing.md })}
-                  >
-                    <ThemeIcon size="sm" mt={2}>
-                      <IconBinaryTree2 />
-                    </ThemeIcon>
-                    <Text size="md" w="90%">
-                      <Text size="sm" color="blue" span mr="md">
-                        {subtask.key}
-                      </Text>
-                      {subtask.fields.summary}{" "}
-                    </Text>
-                  </Paper>
+                  <Subtask
+                    subtaskKey={subtask.key}
+                    id={subtask.id}
+                    fields={subtask.fields}
+                  />
                 ))}
+                <AddSubtask issueKey={issueKey} projectId={projectId} />
               </Stack>
             </Paper>
-          ) : (
-            <Text color="dimmed" mb="xl" fs="italic">
-              Add child Issue
-            </Text>
-          )}
-          <Stack mb={10}>
-            <Text color="dimmed">Comments</Text>
-            {comment.comments.map((commentBody) => (
-              <Paper
-                radius="lg"
-                shadow="sm"
-                withBorder
-                sx={{ minHeight: "10px" }}
-              >
-                <Stack key={commentBody.id} p={10}>
-                  <Group>
-                    <Avatar
-                      src={commentBody.author.avatarUrls["48x48"]}
-                      radius="xl"
+            <CommentSection issueKey={issueKey} comment={comment} />
+          </ScrollArea.Autosize>
+        </Stack>
+        <ScrollArea.Autosize
+          maxHeight="70vh"
+          sx={{ minWidth: "260px", flex: 10 }}
+        >
+          <Box>
+            <Menu shadow="md" position="bottom-start">
+              <Menu.Target>
+                <Button
+                  sx={{
+                    display: "flex",
+                    justifyContent: "flex-start",
+                    flexDirection: "row",
+                  }}
```

Comment:
why does this button have to be a flex itself?

Reply:
??

Reply:
"??" is not a question.
If there is no need for this button to be a flex, let's remove this

Reply:
I did that so that the status text would be aligned from left to right in the button

Reply:
fixed